### PR TITLE
Port standardization

### PR DESCRIPTION
**Pull request by tropicar**
According to this issue https://github.com/dappnode/DAppNode/issues/457
We are selecting the host port for the client.

In prysm client you can change these default ports using these flags:
```
   --p2p-tcp-port value            The port used by libp2p. (default: 13000)
   --p2p-udp-port value            The port used by discv5. (default: 12000)
```

### CODE REVIEW

**Comment by tropicar** [comment]
> there are up to three possible prysm packages (prater eth2 and gnosis chain). How do we handle this situation if the three packages want to use the same hot port?

The port is defined in the next way; chainID multiplied by 10 + default port + clientID(this number is defined by us). You can read the specific feature here https://github.com/dappnode/DAppNode/issues/457

For that specific case:
Prysm: TCP/13103 & UDP/12103
Prysm Prater:  TCP/13503 & UDP/12503
Prysm Gnosis: TCP/14003 & UDP/13003
With the chainID we avoid the collision in the same clients with different networks
Multiply by ten because because of the testnets has the same client and chainIDs have similar values (and how they use the same client, clientID and deffault port are the same ).

File: docker-compose.yml [hunk 1]
```diff
@@ -9,15 +9,17 @@ services:
     volumes:
       - "beacon-chain-data:/data"
     ports:
-      - "12000/udp"
-      - "13000/tcp"
+      - "12004:12004/udp"
+      - "13004:13004/tcp"
```
Comment (alexpeterson91) [inline docker-compose.yml:13]:
```suggestion
      - "14003:14003/tcp"
```
This is the specified port in the port standardization.

File: docker-compose.yml [hunk 2]
```diff
@@ -9,15 +9,17 @@ services:
     volumes:
       - "beacon-chain-data:/data"
     ports:
-      - "12000/udp"
-      - "13000/tcp"
+      - "12004:12004/udp"
```
Comment (alexpeterson91) [inline docker-compose.yml:12]:
```suggestion
      - "13003:13003/udp"
```
this is the specified port.

File: docker-compose.yml [hunk 3]
```diff
@@ -9,15 +9,17 @@ services:
...
+      - "13004:13004/tcp"
     restart: unless-stopped
     environment:
       XDAI_RPC_URL: "https://rpc.gnosischain.com/"
       WEB3_BACKUP: "https://rpc.gnosischain.com/"
       HTTP_WEB3PROVIDER: "http://nethermind-xdai.dappnode:8545"
       CORSDOMAIN: "http://gnosis-beacon-chain-prysm.dappnode"
       DEPLOYMENT_BLOCK: 19469077
+      P2P_TCP_PORT: 13004
```
Comment (alexpeterson91) [inline docker-compose.yml:21]:
```suggestion
      P2P_TCP_PORT: 14003
```

File: docker-compose.yml [hunk 4]
```diff
@@ -9,15 +9,17 @@ services:
...
+      P2P_UDP_PORT: 12004
```
Comment (alexpeterson91) [inline docker-compose.yml:22]:
```suggestion
      P2P_UDP_PORT: 13004
```

**Review by alexpeterson91 — CHANGES_REQUESTED** [review]
not the right ports according to the spec you defined

**Review by alexpeterson91 — APPROVED** [review]

**Review by alexpeterson91 — APPROVED** [review]

**Pull request merged**